packages/form/src/components/RolePicker/RolePickerBasic.tsx

```diff
@@ -39,6 +39,7 @@ export const RolePickerBasic = ({
         optionLabel="name"
         placeholder={placeholder}
         options={options}
+        disabled={options?.length === 1 ? true : false}
```

Comment:
Nice catch. Should we do the same in App picker also?

Reply:
It is done for both, previous PR include for app picker.